Foundation/Summit/2024/index.html

```diff
@@ -20,14 +20,13 @@ <h4>
   <ul>
     <!--li><a href="#summit-important-dates">Important Dates</a></li-->
     <li><a href="#summit-about">About</a></li>
-    <li><a href="#summit-registration">Register for summit</a></li>
+    <li><a href="#summit-registration">Registration</a></li>
     <li><a href="#summit-location">Location & Venue</a></li>
     <li><a href="#summit-cfp">Submissions and Call for Presentations</a></li>
+    <li><a href="#summit-program">Program & Abstracts</a></li>
     <li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
     <!-- li><a href="#summit-panellists">Panellists</a></li>-->
-    <li><a class="highlight" href="#summit-program">Latest news: Program & Abstracts</a></li>
-    <!--li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
-    <li><a href="#summit-panellists">Panellists</a></li-->
+    <li><a class="highlight" href="#summit-training-day">Latest news: Training day @UNSW</a></li>
```

Comment:
```suggestion
    <li><a class="highlight" href="#summit-training-day">Latest news: Training day @ UNSW</a></li>
```

should this have a space between the "@" and "UNSW"?